tuna/miopen/miopen_lib.py

```diff
@@ -176,6 +188,24 @@ def parse_args(self):
     if (self.args.update_applicability or has_fin) and not self.args.session_id:
       parser.error("session_id must be specified with this operation")
 
+  def overwrite_common_args(self):
+    """Overwrite common MIOpen_lib args with subcommand args"""
+    if self.args.subcommand is not None:
+      subc_dict = vars(self.args.get(self.args.subcommand))
+      for sub_key in subc_dict:
+        if sub_key in vars(self.args):
+          self.args[sub_key] = subc_dict.get(sub_key)
+
+  def set_import_cfg_batches(self):
```

Comment:
Put next to parsing in parse_miopen_args?

Reply:
yeah ill move